### DISCUSSION TITLE
Image upload size limit?

### DISCUSSION THREAD

**Discussion by Entheologian** [Dan6erbond/share-me#21] [Q&A] [answered]
Hi,

Is there an image upload size limit / restriction? When I tried a 6.5MB one, it didn't work (no errors) and when I tried 4.5MB it worked. 

Thanks,

**Comment by Dan6erbond** [comment, marked as answer]
Hi, yes. As per the [README](https://github.com/Dan6erbond/share-me#file-size-limit), PocketBase sets a default file size limit of 5MB that I consider a reasonable default. This can be increased by going to the `files` collection > Settings `file` column and increasing the max file size value.

**Reply by Entheologian** [reply]
lol, I guess I missed that... Thanks for clarifying that!
